### DISCUSSION TITLE
HTML or JSON over the wire ?

### DISCUSSION THREAD

**Discussion by Malix-Labs** [unpoly/unpoly#670] [Q&A] [answered]
Does unpoly works with JSON or HTML over the wire ?

**Comment by adam12** [comment, marked as answer]
Unpoly uses HTML over the wire.

**Reply by Malix-Labs** [reply]
Okay, thanks !

Btw, I like HTML-over-the-wire generally, but sometimes I need an endpoint both for a website and a JSON API,
Do you know if this would be possible with unpoly ?

**Reply by adam12** [reply]
Nothing stopping you from mixing and matching HTML and JSON but Unpoly typically doesn't concern itself with JSON responses.
